src/Humanizer.Tests.Shared/Bytes/ToFullWordsTests.cs

```diff
@@ -25,6 +25,7 @@
 
 namespace Humanizer.Tests.Bytes
 {
+    [UseCulture("en")]
```

Comment:
I had to change this to use the `en` culture, because the tests otherwise don't run on a system running in a `de` culture setting.